.eslintrc

```diff
@@ -18,7 +18,8 @@
     "plugin:jsx-a11y/recommended"
   ],
   "rules": {
-    "jsx-quotes": [2, "prefer-double"]
+    "jsx-quotes": [2, "prefer-double"],
+    "jsx-a11y/no-onchange": "off",
```

Comment:
I added that as I looked into the onChange thing and it's 3 use cases that it shouldn't be used for, non of which we are likely to do and with onblur it only changed when I clicked into next form field, not after select so not very useful to us

Reply:
Cool, thank you for looking into that!